modules/serverless-admin-monitoring-eventing-sources.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * serverless/admin_guide/serverless-admin-monitoring.adoc
+
+[id="serverless-admin-monitoring-eventing-sources"]
+= Monitoring event sources
+
+You can use the {product-title} monitoring dashboards to view metrics for event sources in your cluster.
+
+.Prerequisites
+
+* You have cluster administrator permissions, and access to the *Administrator* perspective in the {product-title} web console.
+* You have installed the {ServerlessOperatorName}, as well as the Knative Eventing component.
+* The {product-title} monitoring stack is enabled on your cluster. You can enable monitoring for {ServerlessProductName} during installation, by checking the box to *Enable operator recommended cluster monitoring on this namespace* when installing the {ServerlessOperatorName}.
+
+.Procedure
+
+. In the *Administrator* perspective, navigate to *Monitoring* -> *Dashboards*.
+. Select the *Knative Eventing - Sources* dashboard in the *Dashboard* drop-down.
+. You can now view the following metrics:
+.. For API server source:
+*** Event Count (rate per minute)
+*** Success Rate (2xx Event, fraction rate per minute)
+*** Event Count by Response Code Class (rate per minute)
+*** Failure Rate (non-2xx Event, fraction rate per minute)
+.. For ping source:
+*** Event Count (rate per minute)
+*** Success Rate (2xx Event, fraction rate per minute)
+*** Event Count by Response Code Class (rate per minute)
+*** Failure Rate (non-2xx Event, fraction rate per minute)
+.. For Kafka source:
```

Comment:
Good call!

Reply:
I've actually updated it to be "sources" instead since it's not necessarily a single source, there could be multiple sources of the same type.
Wrt calling it "sources" here instead of event sources; these as a whole are referred to as event sources, and in an ideal world we'd use ApiServerSource, PingSource and KafkaSource here, since these are the proper names, but due to the guidelines for OCP docs I kinda have to write them out this way if I don't want to put object or resource after the word each time.
I wonder if we can start treating these like ConfigMap or other similar API objects instead and just use PingSource etc in the docs without the object/resource bit? I think this is also done for ServiceMeshControlPlane. Might need some guidance from @vikram-redhat on this?